packages/devtools_app/lib/src/debugger/flutter/scripts.dart

```diff
@@ -98,34 +104,50 @@ class ScriptPickerState extends State<ScriptPicker> {
           if (!_isLoading)
             Expanded(
               child: ListView.builder(
-                itemCount: _filteredScripts.length,
+                itemCount: _filteredItems.length,
                 itemExtent: defaultListItemHeight,
                 itemBuilder: (context, index) =>
-                    _buildScript(_filteredScripts[index]),
+                    _buildItemWidget(_filteredItems[index]),
               ),
             ),
         ],
       ),
     );
   }
 
-  Widget _buildScript(ScriptRef ref) {
+  Widget _buildItemWidget(ObjRef ref) {
+    String text;
+    IconData icon;
+
+    if (ref is ScriptRef) {
+      text = ref.uri;
+      icon = libraryIcon;
+    } else if (ref is ClassRef) {
+      text = ref.name;
+      icon = classIcon;
+    }
+
```

Comment:
can ref be anything except the above two classes?

Reply:
no, but I'll add an assert